src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -878,6 +891,16 @@ public function productHasVariant(ProductInterface $product, string $productVari
         $this->createProductVariant($product, $productVariantName, $priceValue, $code, $channel, null, true, $currentStock);
     }
 
+    /**
+     * @Given /^(this product) is disabled in ("[^"]+" channel)$/
+     */
+    public function thisProductIsDisabledInChannel(ProductInterface $product, ChannelInterface $channel): void
+    {
+        $product->removeChannel($channel);
```

Comment:
There is already a method with this behaviour in `ProductContext` `thisProductIsAlsoUnavailableInChannel`